osu.Game/Screens/Ranking/Expanded/ExpandedPanelMiddleContent.cs

```diff
@@ -316,5 +301,49 @@ private void updateDisplay()
                     time.ToLocalTime().ToLocalisableString(prefer24HourTime.Value ? @"d MMMM yyyy HH:mm" : @"d MMMM yyyy h:mm tt"));
             }
         }
+
+        internal class ClickableMetadata : OsuHoverContainer
+        {
+            [Resolved]
+            private OsuGame? game { get; set; }
+
+            public ClickableMetadata(int beatmapId, IBeatmapMetadataInfo metadata)
+            {
+                RelativeSizeAxes = Axes.X;
+                AutoSizeAxes = Axes.Y;
+
+                Anchor = Anchor.TopCentre;
+                Origin = Anchor.TopCentre;
+
+                Child = new FillFlowContainer
+                {
+                    RelativeSizeAxes = Axes.X,
+                    AutoSizeAxes = Axes.Y,
```

Comment:
Should be autosized x to fix this:

<img width="350" alt="Screenshot 2025-03-12 at 10 53 46 PM" src="https://github.com/user-attachments/assets/2680d137-089c-4d10-b53b-9002274c1456" />

```suggestion
                AutoSizeAxes = Axes.Both;

                Anchor = Anchor.TopCentre;
                Origin = Anchor.TopCentre;

                Child = new FillFlowContainer
                {
                    AutoSizeAxes = Axes.Both,
```